### PR TITLE
Bump prettier to version 3.3.3

### PR DESCRIPTION
**Pull request by latekvo**
## Description

Bump prettier version to 3.3.3.
Required for [this](https://github.com/software-mansion/react-native-gesture-handler/pull/3146#discussion_r1829571748) issue to be resolved.

## Test plan

- confirm all CIs are passing
- confirm none of the examples are crashing

### CODE REVIEW

File: src/handlers/gestureHandlerCommon.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ export interface GestureEvent<ExtraEventPayloadT = Record<string, unknown>> {
   nativeEvent: Readonly<GestureEventPayload & ExtraEventPayloadT>;
 }
 export interface HandlerStateChangeEvent<
-  ExtraEventPayloadT = Record<string, unknown>
+  ExtraEventPayloadT = Record<string, unknown>,
```
[the remark this answers is not in the document]
Comment (latekvo) [reply src/handlers/gestureHandlerCommon.ts:145]: Sure, but that would bump the changed files count from 16 to over 100, is that still ok?
I'll also have to change the eslint config which i think starts to get a bit out of scope of this PR.

[the remark this answers is not in the document]
Comment (m-bert) [reply src/handlers/gestureHandlerCommon.ts:145]: Okay, in that case let's leave for now. We can create separate PR for that later.

**Review by m-bert — APPROVED** [review]
I don't like those commas that I've mentioned earlier, but we can remove them for example in GH3 😅

**Comment by latekvo** [comment]
> we can remove them [...] in GH3

Sure

**Pull request merged**